src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -13,6 +16,16 @@ const StyledField = styled(Field)`
   }
 `;
 
+const StyledIconButton = styled(IconButton)`
+  position: absolute;
+  top: 4px;
+  right: 4px;
+`;
+
+const StyledMediaInput = styled(MediaInput)`
+  padding-right: 22px;
```

Comment:
I think we should add a bit more space, maybe 4px, the padding is just the minimum needed

<img width="424" alt="Screenshot 2025-03-26 at 17 57 48" src="https://github.com/user-attachments/assets/1ba8cac4-f388-4ae5-a4a1-0a3b7e56f60e" />